### PR TITLE
PiiScanner Add Word Boundary Check

### PR DESCRIPTION
**Pull request by kingthorin**
- PiiScanner add word boundary checks to reduce false positives.
- Added unittest and a utility method for message creation.
- Updated manifest.
- Fixed typo in `MAESTRO` enum name.

https://www.regexplanet.com/share/index.html?share=yyyyyq6d1nr (Click java, then test)

Fixes zaproxy/zaproxy#5124

### CODE REVIEW

**Comment by kingthorin** [comment]
Working on it...

**Comment by kingthorin** [comment]
I must be missing something subtle, the new unittest works as expected, however the original parameterized version doesn't. I can't see why, regexplanet indicates things should be fine with the boundaries.

**Comment by kingthorin** [comment]
If I switch to using `^` and `$` markers instead of word boundaries then both unittests work, does that make sense?

**Comment by kingthorin** [comment]
Using `^`and `$` gets the Unit tests all passing but doesn't provide as good coverage of matches per regexplanet.

**Comment by kingthorin** [comment]
UGH ok well I'm going to document it so others can learn from my mistake:
image

Make sure you use the "Java string" version. `\\b` not `\b`...

File: src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
 	<url/>
 	<changes>
 	<![CDATA[
+		PiiScanner add word boundry checks to reduce false positives.<br>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml]: @kingthorin the typo was here as well (just noticed in the migration), mind opening a PR?

Comment (kingthorin) [reply src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml]: On it.

**Pull request merged**